_posts/2020-08-21-running-tutorials.md

```diff
@@ -0,0 +1,340 @@
+
+We also want to ensure folks are taking things in, and labs are a great way to include small tests in the content. Having a section at the end of an example which is incomplete means that you can give the audience some time to try and figure things our for themselves. Some folks will be able to fill things in with no problems. For others they will hit errors or make mistakes, this is good for teaching how to debug and troubleshoot your project. And for those who are having awful flashbacks to pop-quizzes they can simply skip it without worrying that someone will check up on them.
+
+For each section of content you want to include in your tutorial I recommend you create a notebook with an explanation, an example and some things for the audience to figure out. Doing this for each section (in the Dask tutorial we had 9 sections) the audience will quickly become familiar with the process and be able to anticipate what is coming next. This will make them feel comfortable.
+
+### Hosting the material
+
+Once you have put your material together you need to share it with your attendees.
+
+GitHub is a great place to put things, especially if you include an open license with it. For narrative tutorial content a creative commons license if often used which requires modifications to also be shared.
+
+As we have put our content together as Jupyter Notebooks we can use [Binder](https://mybinder.org/) to make it possible for folks to run the material without having to download it locally or ensure their Python environment is set up correctly.
+
+### Choosing a video platform
+
+Next we have to decide how we will present the material. As this is a virtual tutorial we will want to use some kind of video conferences or streaming software.
+
+These tools tend to fall into two categories; private meetings with a tool like Zoom, Hangouts or Teams and public broadcasts on websites like YouTube or Twitch.
+
+Any of these options will likely be a good choice, they allow the presenter to share their video, audio and screen with participants and participants can communicate back with a range of tools.
+
+The main decision you will have to make is around whether you want to restrict numbers or not. The more interactivity you want to have in the tutorial the more you will need a handle on numbers. For our initial tutorials we wanted to enable participants to ask questions at any time and get a quick response, so we opted to use Zoom and limit our numbers to allow us to not get overwhelmed with questions. However if you want to present to as many people as possible and accept that you may not be able to address them all individually you may want to use a streaming platform instead.
```

Comment:
It may be worth noting that you can do both at the same time.  Zoom does allow streaming to Youtube as well.  This is what we do with Coiled Science Thursdays and I think also what RAPIDS-Academy does.  It works out pretty well.

Reply:
Also, for Zoom I think that we set some option like "Mute attendees on entering"

Reply:
Good points. I've addressed these in 7c488e2.